css/properties/text-underline-offset.json

```
@@ -6,8 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-underline-offset",
           "support": {
             "chrome": {
-              "version_added": false,
-              "notes": "See Chrome Platform Status entry for <a href='https://chromestatus.com/feature/5747636764147712'>text-decoration-thickness, text-underline-offset and from-font keyword for text-underline-position</a>."
+              "version_added": "87"
             },
             "chrome_android": {
               "version_added": false
```

```suggestion
              "version_added": "87"
```

The same would go for `webview_android` below

---

Done